app/index.js

```diff
@@ -83,6 +83,18 @@ var AngularFullstackGenerator = yeoman.generators.Base.extend({
 
       this.prompt([{
           type: 'list',
+          name: 'buildtool',
+          message: 'Would you like to use Gulp (experimental) instead of Grunt?',
+          choices: [ 'Grunt', 'Gulp', 'Both'],
+          filter: function(val) {
+            return {
+              'Grunt': 'grunt',
+              'Gulp': 'gulp',
+              'Both': 'grunt_and_gulp'
+            }[val];
+          }
+        }, {
+          type: "list",
```

Comment:
What do you think about moving this to [`projectPrompts`](https://github.com/DaftMonk/generator-angular-fullstack/blob/canary/app/index.js#L244-L296)?